README.md

```diff
@@ -81,15 +81,35 @@ If you need more detailed instructions, see the [Novice setup guide][novice-setu
 [wiki]: https://github.com/cantino/huginn/wiki
 [novice-setup-guide]: https://github.com/cantino/huginn/wiki/Novice-setup-guide
 
+### Using Vagrant
```

Comment:
Should Vagrantfile be used as a development platform or as a get it up an running quick so that one can check out Huginn without knowledge of rails?